### ISSUE TITLE
Make log message less repetitive

### ISSUE THREAD

**Issue by seledwards** [GENI-NSF/geni-tools#385]
The following log message is repetitive:
```
17:29:09 INFO     omni: Config file '/Users/sedwards/gcf2/agg_nick_cache' or '/Users/sedwards/gcf2/agg_nick_cache' does not exist
```

Get it when calling:
```
$ omni.py --AggNickCacheName ~/gcf2/agg_nick_cache  getversion -a ig-gpo
10:05:32 INFO     omni: Config file '/Users/sedwards/gcf2/agg_nick_cache' or '/Users/sedwards/gcf2/agg_nick_cache' does not exist
```

 _Imported from trac ticket [#385](http://trac.gpolab.bbn.com/gcf/ticket/385),  created by sedwards on 09-25-2013 at 10:14, last modified: 02-13-2014 at 08:32_
   CCing: gpo-expt-support@...

**Comment by seledwards** [comment]
Fixed in the process of fixing #384.

_Trac comment by sedwards (github user: seledwards) on 09-26-2013 at 12:06_

**Comment by luisanevers** [comment]
Verified that the AggNickCacheName not found message is no longer printed.  

In gcf 2.4.1 when the file did not exist the following was reported:
```
$  omni.py --AggNickCacheName ~/mydir/agg_nick_cache  getversion -a gpo-ig
08:29:29 INFO     omni: Config file '/home/lnevers/mydir/agg_nick_cache' or '/home/lnevers/mydir/agg_nick_cache' does not exist
08:29:29 INFO     omni: Attempted to download latest `agg_nick_cache` from 'http://trac.gpolab.bbn.com/gcf/raw-attachment/wiki/Omni/agg_nick_cache' but could not.
08:29:29 INFO     omni: Loading agg_nick_cache file '/home/lnevers/.gcf/agg_nick_cache'
08:29:29 INFO     omni: Loading config file /home/lnevers/.gcf/omni_config
```
The same scenario in gcf-2.5-021214pm no longer prints the repetive message:
```
$ omni.py --AggNickCacheName ~/mydir/agg_nick_cache  getversion -a gpo-ig -o
08:31:06 INFO     omni: Downloaded latest `agg_nick_cache` from 'http://trac.gpolab.bbn.com/gcf/raw-attachment/wiki/Omni/agg_nick_cache' and copied to '/home/lnevers/mydir/agg_nick_cache'.
08:31:06 INFO     omni: Loading agg_nick_cache file '/home/lnevers/mydir/agg_nick_cache'
08:31:06 INFO     omni: Loading config file /home/lnevers/.gcf/omni_config
```


_Trac comment by lnevers (github user: luisanevers) on 02-13-2014 at 08:32_
